components/popover/popover-mixin.js

```diff
@@ -100,19 +106,41 @@ export const PopoverMixin = superclass => class extends superclass {
 			this._previousFocusableAncestor = this.opened ? getPreviousFocusableAncestor(this, false, false) : null;
 
 			if (this.opened) {
+
 				this._opener = getComposedActiveElement();
 				this._addAutoCloseHandlers();
-				this._dismissibleId = setDismissible(() => this._close());
+				this._dismissibleId = setDismissible(() => this.close());
+				this._focusContent(this);
 				this.dispatchEvent(new CustomEvent('d2l-popover-open', { bubbles: true, composed: true }));
+
 			} else if (changedProperties.get('opened') !== undefined) {
+
 				this._removeAutoCloseHandlers();
 				this._clearDismissible();
+				this._focusOpener();
 				this.dispatchEvent(new CustomEvent('d2l-popover-close', { bubbles: true, composed: true }));
+
 			}
 
 		}
 	}
 
+	close() {
+		this.opened = false;
+		return this.updateComplete;
+	}
+
+	open(applyFocus = true) {
+		this._applyFocus = applyFocus !== undefined ? applyFocus : true;
```

Comment:
If `opened` is true on initial load (or a consumer opens the popover by setting `opened` instead of using `open()`) then `this._applyFocus` wouldn't necessarily be true. Is this okay?

Reply:
My plan is to actually remove the `opened` public property altogether. The mixin consumer will be required to use the methods to control the state. But aside from that, yes... it default to applying the focus. The only case where it would not is when the dropdown is opened via hovering, and dropdown uses the methods to toggle the open state for that reason.